pkg/cassandra/version/version.go

```diff
@@ -2,11 +2,9 @@ package version
 
 import (
 	"encoding/json"
-	"fmt"
 	"strconv"
-	"strings"
 
-	"github.com/coreos/go-semver/semver"
+	semver "github.com/hashicorp/go-version"
 )
 
```

Comment:
Can we move this file out of `pkg/cassandra` and into something like `pkg/api/util`? This is an API type, but it isn't by default part of an API group (hence it isn't in apis).

It'd be good to denote it as such in the file structure, to make it clear this isn't specific to just cassandra (and also denotes that this type may be exposed in an api)

Reply:
Done.